homework/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+
+        <meta charset="UTF-8">
+        <title>HUDA ALISMAIL</title>
+        <link rel="stylesheet" href ="styles/styles.css">
+
+    <body>
+
```

Comment:
when using footer and header tags inside body be sure to use the main tag to help you separate the main content of your page from the header and footer.